src/main/java/org/jenkinsci/plugins/parallel_test_executor/PreviousTestResultLookup.java

```diff
@@ -0,0 +1,61 @@
+package org.jenkinsci.plugins.parallel_test_executor;
+
+import com.google.common.base.Predicate;
+import hudson.ExtensionPoint;
+import hudson.model.Run;
+import hudson.tasks.test.TestResult;
+import org.jenkinsci.plugins.variant.OptionalExtension;
+import org.jenkinsci.plugins.workflow.cps.nodes.StepStartNode;
```

Comment:
No, use `BlockStartNode & `org.jenkinsci.plugins.workflow.graph.StepNode` so you do not need a `workflow-cps` dep.

Reply:
Okiedokie